tests/Replication2/ReplicatedLog/SimpleInsertTests.cpp

```diff
@@ -45,10 +45,8 @@ TEST_F(ReplicatedLogTest, write_single_entry_to_follower) {
   auto follower = std::make_shared<DelayedFollowerLog>(
       defaultLogger(), _logMetricsMock, followerId, std::move(coreB),
       LogTerm{1}, leaderId);
-  auto leader = LogLeader::construct(
-      LoggerContext(Logger::REPLICATION2), _logMetricsMock, _optionsMock,
-      leaderId, std::move(coreA), LogTerm{1},
-      std::vector<std::shared_ptr<AbstractFollower>>{follower}, 2);
+  auto leader =
+      createLeader(leaderId, LogTerm{1}, std::move(coreA), {follower}, 2);
```

Comment:
Sorry for being picky but maybe we want to call the function `createLeaderWithDefaultFlags`?

Reply:
Yes, that's much better name.